projects/packages/search/src/dashboard/components/record-meter/notice-box.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Returns a notice box for displaying notices about record count and plan limits
+ *
+ * @param {object} props - record count, record limit, hasbeenindexed, hasvaliddata, hasItems
+ * @returns {React.Component} notice box component.
+ */
+export function NoticeBox( props ) {
+	const notices = [];
+	const CLOSE_TO_LIMIT_PERCENT = 0.8; //TODO: currently 'close' is defined as 80%. This has not been decided/finalised to be the best number here yet
```

Comment:
Much better 👍 Let's move this constant out of the function and define it after the imports.